queries.sql

```diff
@@ -122,3 +122,61 @@ COUNT(a.owner_id)
 FROM animals a
 INNER JOIN owners o ON o.id = a.owner_id
 GROUP BY o.full_name;
+
+-- DAY 4
+-- Who was the last animal seen by William Tatcher?
+SELECT vets.name AS vet_name, animals.name AS Last_seen_animal, date_of_visit AS visit_date FROM visits 
+JOIN animals ON animals.id = visits.animals_id 
+JOIN vets ON vets.id = visits.vets_id 
+WHERE vets.id = 1 ORDER BY date_of_visit desc 
+LIMIT 1;
+
+-- How many different animals did Stephanie Mendez see?
+SELECT COUNT(DISTINCT animals.name) FROM animals
+JOIN owners ON animals.owner_id = owners.id
+WHERE owners.full_name = 'Stephanie Mendez';
+
+-- List all vets and their specialties, including vets with no specialties.
+SELECT vets.name AS vet_name, species.name AS species FROM specializations 
```

Comment:
- [ ] I think it would be great if you made here vets.* since here the question to list all the vets not just the name 🔥♻🚀

Reply:
Thank you. The correction has been implemented.